static/js/components/Navigation.tsx

```diff
@@ -101,6 +101,16 @@ const Navigation: FC = () => {
                         Projects
                       </NavLink>
                     </li>
+                    <li className="p-side-navigation__item--title">
+                      <NavLink
+                        className="p-side-navigation__link"
+                        to="/storages"
+                        onClick={toggleMenu}
+                      >
+                        <i className="p-icon--switcher-environments is-light p-side-navigation__icon"></i>{" "}
```

Comment:
We can use the `p-icon--pods` icon for now. For new icons, ensure to include it in `styles.scss`.

I will request a dedicated icon for vanilla, but it may take some time to become available. Until then, let's use the pods icon. Thank you.

Reply:
PS: We use vanilla as CSS framework. The available icons can be found here: https://vanillaframework.io/docs/patterns/icons

Reply:
@edlerd resolved